lib/popup.ts

```diff
@@ -142,6 +147,25 @@ export function setPopupToCreateDom(triggerElem: Element, domCreator: IPopupDomC
   return setPopupToFunc(triggerElem, openFunc, options);
 }
 
+/**
+ * Open a popup using as content the element returned by the given func.
+ */
+export function popupOpen(reference: Element, domCreator: IPopupDomCreator, options: IPopupOptions): PopupControl {
+  function openFunc(openCtl: IOpenController) {
+    const content = domCreator(openCtl);
+    function dispose() { domDispose(content); ctl.dispose(); }
+    return {content, dispose};
+  }
+  const ctl = PopupControl.create(null) as PopupControl;
+  ctl.attachElem(reference, openFunc, {
+    ...options,
+    // Overrides '.trigger' to avoid attaching listeners to reference.
+    trigger: undefined
```

Comment:
Could you add a comment to function documentation that the `trigger` option is ignored by this function?